src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -234,6 +234,9 @@ private static unsafe bool InitializeConfigAndDetermineUsePortableThreadPool()
                 ? AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false)
                 : GetEnableWorkerTrackingNative();
 
+        internal static readonly bool EnableDispatchAutoReleasePool =
```

Comment:
`AutoRelease` -> `Autorelease`? The config name, file names, and P/Invokes use lowercase `r`.